src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+form {
+  margin-top: 20px;
+}
```

Comment:
Don't style elements using tags. Use classes for this. (Exception - resets and `html` or `body` tags).